package.json

```diff
@@ -47,6 +46,6 @@
     }
   ],
   "dependencies": {
-    "@haventec/common-web-sdk": "1.1.2"
+    "@haventec/common-web-sdk": "1.2.4"
```

Comment:
Double check this after common-web-sdk is reviewed & merged from develop to master

Reply:
Versions are aligned now